rotkehlchen/tests/unit/decoders/test_sushiswap.py

```diff
@@ -2,529 +2,235 @@
             location_label=ADDY_1,
-            notes='Burned 0.00393701451 ETH for gas',
+            notes=f'Burned {gas} ETH for gas',
             counterparty=CPT_GAS,
         ), EvmEvent(
-            tx_hash=evmhash,
-            sequence_index=1,
-            timestamp=TimestampMS(1646375440000),
+            tx_hash=tx_hash,
+            sequence_index=308,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.INFORMATIONAL,
+            event_subtype=HistoryEventSubType.APPROVE,
+            asset=Asset('eip155:1/erc20:0x62B9c7356A2Dc64a1969e19C23e4f579F9810Aa7'),
+            balance=Balance(amount=FVal(approved_amount)),
+            location_label=ADDY_1,
+            notes=f'Set cvxCRV spending approval of {ADDY_1} by 0xd9e1cE17f2641f24aE83637ab66a2cca9C378B9F to {approved_amount}',  # noqa: E501
+            counterparty=None,
+            address=string_to_evm_address('0xd9e1cE17f2641f24aE83637ab66a2cca9C378B9F'),
+        ), EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=309,
+            timestamp=timestamp,
             location=Location.ETHEREUM,
             event_type=HistoryEventType.TRADE,
             event_subtype=HistoryEventSubType.SPEND,
             asset=EvmToken('eip155:1/erc20:0x62B9c7356A2Dc64a1969e19C23e4f579F9810Aa7'),
-            balance=Balance(amount=FVal('19.157411925828275084'), usd_value=ZERO),
+            balance=Balance(amount=FVal(swap_amount)),
             location_label=ADDY_1,
-            notes='Swap 19.157411925828275084 cvxCRV in sushiswap-v2 from 0x3Ba6eB0e4327B96aDe6D4f3b578724208a590CEF',  # noqa: E501
+            notes=f'Swap {swap_amount} cvxCRV in sushiswap-v2 from {ADDY_1}',
             counterparty=CPT_SUSHISWAP_V2,
             address=string_to_evm_address('0x33F6DDAEa2a8a54062E021873bCaEE006CdF4007'),
         ), EvmEvent(
-            tx_hash=evmhash,
-            sequence_index=2,
-            timestamp=TimestampMS(1646375440000),
+            tx_hash=tx_hash,
+            sequence_index=310,
+            timestamp=timestamp,
             location=Location.ETHEREUM,
             event_type=HistoryEventType.TRADE,
             event_subtype=HistoryEventSubType.RECEIVE,
             asset=EvmToken('eip155:1/erc20:0xD533a949740bb3306d119CC777fa900bA034cd52'),
-            balance=Balance(amount=FVal('18.47349725628421943'), usd_value=ZERO),
+            balance=Balance(amount=FVal(received_amount)),
             location_label=ADDY_1,
-            notes='Receive 18.47349725628421943 CRV in sushiswap-v2 from 0x3Ba6eB0e4327B96aDe6D4f3b578724208a590CEF',  # noqa: E501
+            notes=f'Receive {received_amount} CRV in sushiswap-v2 from {ADDY_1}',
             counterparty=CPT_SUSHISWAP_V2,
             address=string_to_evm_address('0x33F6DDAEa2a8a54062E021873bCaEE006CdF4007'),
         ),
     ]
     assert events == expected_events
 
 
+@pytest.mark.vcr()
```

Comment:
Should not make a huge difference, but try to be putting the api key filter here